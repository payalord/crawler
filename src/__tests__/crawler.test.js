import Crawler from '../crawler'
jest.mock('../libraries/request')

describe('Crawler test', () => {
    const url = 'https://localhost:3000/'
    it('Response should be an html document', async () => {
        const result = await Crawler(url)
        expect(result).toMatchSnapshot()
    })
})