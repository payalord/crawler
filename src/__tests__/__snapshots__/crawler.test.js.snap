// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Crawler test Response should be an html document 1`] = `
"<!DOCTYPE html>
    <html>
    <body>
    <a href=\\"https://localhost:3000/test1\\">Test Page 1</a>
    <a href=\\"https://localhost:3000/test2\\">Test Page 2</a>
    <a href=\\"https://example.com/test3\\">Example of External link</a>
    <img src=\\"https://localhost:3000/image1.png\\" />
    <img src=\\"https://localhost:3000/image2.png\\" />
    </body>
    </html>"
`;
